### PR TITLE
Add vagababov to eng-approvers and sort and trim people not longer in Knative project

### PR DESCRIPTION
**Pull request by vagababov**
Also add `eng-approvers` as approvers to the serving docs.
Since that seems the reason for this group in the first place.

/assign mattmoor @abrennan89

### CODE REVIEW

File: OWNERS_ALIASES
Comment (mattmoor) [inline OWNERS_ALIASES:2]: IDK why we have both eng-approvers and serving-eng-reviewers, this seems.... complex.

@abrennan89 WDYT about simplifying things?

Comment (vagababov) [reply OWNERS_ALIASES:2]: I can merge both lists.
THough usually we have both everywhere.

**Review by abrennan89 — APPROVED** [review]
/lgtm
/hold

I'm fine with this either way @mattmoor :slightly_smiling_face: 

Since this list isn't really maintained by docs and just happens to live here I'd say it's up to the Serving WG how they want to proceed. Gonna go ahead and approve but I'll put the hold on this so you can figure out what you want to do and remove it when you want to merge.

File: OWNERS_ALIASES
Comment (vagababov) [reply OWNERS_ALIASES:2]: Merged the two

**Review by mattmoor — COMMENTED** [review]
We could go further, but this seems fine for now.
/unhold
/lgtm

**Pull request merged**
